addon/components/table-column.js

```diff
@@ -13,6 +13,7 @@ const {
 export default Component.extend({
   layout,
   tagName: 'td',
+  attributeBindings: ['title'],
```

Comment:
lets make this `attributeBindings: ['cellTitle:title']`and add the changes below.